apps/wpcom-block-editor/package.json

```diff
@@ -16,15 +16,19 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
-		"clean": "npx rimraf dist",
-		"prebuild": "npm run clean",
-		"bundle": "webpack --config='../../packages/calypso-build/webpack.config.js' --env.WP='true' iframe-bridge-server='./src/iframe-bridge-server.js' tinymce='./src/tinymce.js'",
-		"build": "npm run bundle -- --output-path=./dist"
+		"bundle": "webpack --config='../../packages/calypso-build/webpack.config.js' --env.WP='true'",
+		"build:common": "npm run bundle -- common='./src/common'",
+		"build:calypso-iframe-bridge-server": "npm run bundle -- calypso-iframe-bridge-server='./src/calypso/iframe-bridge-server.js'",
+		"build:calypso-tinymce": "npm run bundle -- calypso-tinymce='./src/calypso/tinymce.js'",
+		"build": "npm-run-all --parallel \"build:* -- {@}\" --"
```

Comment:
If this gets too funky at some point, you could also create a simple Webpack config for this package that inherits and extends the one from Calypso-build, similarly to `client/notifications/`.